src/clusterfuzz/_internal/datastore/data_handler.py

```diff
@@ -571,6 +594,22 @@ def get_issue_description(testcase,
   return content_string
 
 
+def get_github_issue_title(testcase):
```

Comment:
Same with all the changes in this file. Let's put this all in github.py. It's better to keep all the related logic in one place.